### PR TITLE
update docs with new status output

### PR DESCRIPTION
**Pull request by BelSasha**

### CODE REVIEW

**Comment by BelSasha** [comment]
* **#1242** <a href="https://app.graphite.dev/github/pr/run-house/runhouse/1242?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @BelSasha and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: docs/tutorials/quick-start-local.rst [hunk 1]
```diff
@@ -39,28 +39,28 @@ and details of the server. For printing cluster's status outside the cluser, its
...
     • _cluster_default_env (runhouse.Env)
-      This environment has only python packages installed, if such provided. No resources were found.
-    • sd_env (runhouse.Env) | pid: 29716 | node: head (52.207.212.159)
-      CPU: 0.0% | Memory: 0.13 / 8 Gb (1.65%)
-      This environment has only python packages installed, if such provided. No resources were found.
-    • np_pd_env (runhouse.Env) | pid: 29578 | node: head (52.207.212.159)
-      CPU: 0.0% | Memory: 0.13 / 8 Gb (1.71%)
-      • /sashab/summer (runhouse.Function)
-      • mult (runhouse.Function)
+        This environment has only python packages installed, if such provided. No resources were found.
```
Comment (jlewitt1) [inline docs/tutorials/quick-start-local.rst:57]: can we add to this PR changing the text from "if such provided" to "if provided"?

File: runhouse/main.py [hunk 2]
```diff
@@ -222,13 +222,7 @@ def _print_cluster_config(cluster_config: Dict):
...
     ]
 
-    backend_config = [
-        "resource_subtype",
-        "domain",
-        "server_host",
-        "ips",
-        "resource_subtype",
-    ]
+    backend_config = ["resource_subtype", "domain", "server_host", "ips"]
```
Comment (jlewitt1) [inline runhouse/main.py:225]: why is resource_subtype removed?

Comment (BelSasha) [reply runhouse/main.py:225]: Because it was added twice to the `backend_config` list, by mistake. Not it appears only once.

**Pull request merged**